lua/ui/lobby/lobby_modules/createui.lua

```
@@ -0,0 +1,1859 @@
+                    end
+                end
+            else
+                manualTeams = true
+            end
+
+            -- If the teams were not set properly, set them properly.
+            -- When teams are set manually, they are not set properly if the number of 
+            -- players on either team does not equal the team size.  
+            -- When teams are not set manually, they are not set properly if the number
+            -- of slots on either team is less than the team size.
+            if (manualTeams and (numPlayersTeam1 != teamSize or numPlayersTeam2 != teamSize))
```

Luacheck (1.2.0)

[error] expected ')' near '!'

(E011)